src/shared/internal-calendar/helper.ts

```diff
@@ -0,0 +1,29 @@
+import dayjs from "dayjs";
+
+// ===========================================================================
+// HELPER FUNCTIONS
+// ===========================================================================
+
+export namespace InternalCalendarHelper {
```

Comment:
there is an existing `calendar-helper` under the util folder, this helper can go there